src/app/window.js

```diff
@@ -216,9 +216,8 @@ const MainWindow = new Lang.Class({
                         'Tech Haven Ministries',
                         'Jim Pennucci' ];
 
-        let copyright = 'Copyright 2013-2015 The Weather Developers';
         let attribution = this._cityView.info ? this._cityView.info.get_attribution() : '';
-        let copyright = copyright + (attribution ? '\n' + attribution : '');
+        let copyright = 'Copyright 2013-2015 The Weather Developers' + (attribution ? '\n' + attribution : '');
```

Comment:
Can you cherry-pick this commit [1] instead, or just directly update our copy to the 3.24 version?

[1] https://git.gnome.org/browse/gnome-weather/commit/?id=39c65724bef050561fb605f29019bc60669710ec